dyn_em/solve_em.F

```diff
@@ -222,6 +262,19 @@ SUBROUTINE solve_em ( grid , config_flags  &
 ! Initialize timers if compiled with -DBENCH
 #include "bench_solve_em_init.h"
 
+#if (WRF_CMAQ == 1)
+   if (firstime) then
+      CALL nl_get_feedback_restart ( .false., feedback_restart )
+      if (feedback_restart) then
+         feedback_is_ready = .true.
+      else
+         feedback_is_ready = .false.
+      end if
+   end if
+#else
+   feedback_is_ready = .false.
+#endif
+
```

Comment:
@dwongepa 
These sorts of "if this is the first time step" initializations should be processed by dyn_em/start_em.F (or the associated dyn_nmm file) or in the phys/module_physics_init.F file.

Reply:
I just looked further down, so perhaps you need to explain to us what happens with this switch.